packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/tasks/tasks.html

```diff
@@ -0,0 +1,96 @@
+    data-on-click="$ctrl.goBack()"
+></oui-back-button>
+
+<div class="d-flex justify-content-between align-items-center mb-4">
+    <div>
+        <h2
+            class="mb-0"
+            data-translate="pci_databases_connectors_tasks_title"
+            data-translate-values="{name: $ctrl.connector.name}"
+        ></h2>
+        <span class="oui-paragraph">
+            <span>Type: {{$ctrl.connector.connectorInformation.name}}</span>
+            <span>
+                | Version:
+                {{$ctrl.connector.connectorInformation.version}}</span
+            >
+            <span>
+                | Author: {{$ctrl.connector.connectorInformation.author}}</span
+            >
+            <span ng-if="$ctrl.connector.connectorInformation.documentationUrl">
+                |
+                <a
+                    ng-href="{{$ctrl.connector.connectorInformation.documentationUrl}}"
+                    target="_blank"
+                    rel="noopener"
+                    >docs</a
+                >
+            </span>
+        </span>
+    </div>
+    <oui-button
+        variant="primary"
+        size="l"
+        ng-click="$ctrl.goToEdit($ctrl.connector)"
+    >
+        <span
+            data-translate="pci_databases_connectors_tasks_modify_configuration"
+        ></span>
+    </oui-button>
+</div>
+
+<!-- error messages -->
+<cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+
+<oui-datagrid rows="$ctrl.connector.tasks">
+    <oui-datagrid-column
+        data-title="::'pci_databases_connectors_tasks_id' | translate"
+        property="id"
+        type="number"
+        data-sortable
+    >
+        <span
+            data-translate="pci_databases_connectors_tasks_id_number"
+            data-translate-values="{id: $row.id}"
+        ></span>
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title="::'pci_databases_connectors_tasks_trace' | translate"
+        property="trace"
+        type="string"
+        data-sortable
+        data-searchable
+    >
+        <text-accordion value="$row.trace"></text-accordion>
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title=":: 'pci_databases_connectors_tasks_status' | translate"
+        data-property="status"
+        data-sortable="asc"
+        data-type="string"
+        data-searchable
+        data-filterable
+    >
+        <span
+            class="oui-badge"
+            data-ng-class="{
+                'oui-badge_success': $row.status === $ctrl.TASK_STATUS.RUNNING,
+                'oui-badge_warning':  $row.status === $ctrl.TASK_STATUS.PAUSED,
+                'oui-badge_error':  $row.status === $ctrl.TASK_STATUS.FAILED,
+            }"
+        >
+            <span>{{$row.status}}</span>
+        </span>
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title="::'pci_databases_connectors_tasks_restart' | translate"
+        data-sortable
+    >
+        <restart-button
```

Comment:
**suggestion**: Buttons should be avoided on the grid. `oui-action-menu` should be used instead.

Reply:
Hi @anooparveti , we really need to have the buttons in the table here.
It is something we have already done before  (on indexes for exemple: https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/storages/databases/database/indexes/indexes.html#L31).
I had also checked with Carl Lambert, who confirmed that it is something we can do in the datagrid component.

Reply:
Hello @jperchoc ,
In that case it's advisable to use `oui-button`.